templates/openstack/consulting.html

```diff
@@ -122,7 +122,7 @@ <h2 class="p-heading--3">Accelerate your time-to-market with Private Cloud Build
       <div class="col-6">
         <h2>OpenStack implementation challenges</h2>
         <p>
-          Designing an optimal architecture for OpenStack implementation that maximises price-performance requires expert knowledge and years of experience.
+          Designing an optimal architecture for OpenStack that maximises price performance isn't easy on your own. It requires years of experience and expert knowledge.
```

Comment:
```suggestion
          Designing an optimal architecture for OpenStack that maximizes price performance isn't easy on your own. It requires years of experience and expert knowledge.
```
"maximises" is BE, not AE, but I imagine you may want to tackle that as part of a separate item of work than this issue. Just highlighting